scan-framework/src/main/java/com/getbouncer/scan/framework/util/Layout.kt

```diff
@@ -81,4 +81,92 @@ fun RectF.centerScaled(scaleX: Float, scaleY: Float): RectF {
     )
 }
 
+/**
+ * Scales up a rect
+ */
+fun Rect.scale(scaleX: Float, scaleY: Float) = Rect(
+    this.centerX() - (this.width() * scaleX / 2).toInt(),
+    this.centerY() - (this.height() * scaleY / 2).toInt(),
+    this.centerX() + (this.width() * scaleX / 2).toInt(),
+    this.centerY() + (this.height() * scaleY / 2).toInt()
+)
```

Comment:
this is essentially the same as `RectF.centerScaled`. Let's rename this to `Rect.centerScaled` to match and move it next to `RectF.centerScaled`